services/helper.py

```diff
@@ -8,9 +7,28 @@ def __init__(self, repository: HelperRepository, logger: logging.Logger):
         self.logger = logger
         self.repository = repository
 
-    def handle_thankyou(self, message: discord.Message):
-        thanked_users: list[discord.Member] = message.mentions
-        for user in thanked_users:
-            self.repository.add_points_to_user(user.id, 1)
-            self.logger.info(f'User {user.display_name} got {1} point from '
-                             f'{message.author.display_name} for being helpful')
+    def add_points(self, thanked, thanker, bot, p):
+        if thanked not in {thanker, bot}:
+            self.repository.add_points_to_user(thanked.id, p)
+            if p == 1:
+                self.logger.debug(f'User {thanked.display_name} got {p} point from '
```

Comment:
taki bajer: `... got {p} point{"" if p == 1 else "s"} from ...` polecam zamiast tego ifa dużego, niżej to samo
właśnie takie _pythonic way_ bajery miałem na myśli jak zwróciłem na to uwagę w poprzednim PRze :D wiele można w tym pythonie robić co się filozofom nie śniło